### ISSUE TITLE
Join window x64 support, I have compiled it.

### ISSUE THREAD

**Issue by allphfa** [felipenoris/Mongoc.jl#1]
[Mongoc.jl - window version](https://github.com/AlphaFase/Mongoc.jl)
[commit info](https://github.com/AlphaFase/Mongoc.jl/commit/7b88585ee1ce3dbef9f919e9896fe811b02ec501)

Mongo does not support x86, only x64 version

**Comment by felipenoris** [comment]
This is great!
How did you compile it? Do tests pass on windows?

**Comment by AlphaFase** [comment]
[Mongo Official compilation course](http://mongoc.org/libmongoc/1.12.0/installing.html#building-on-windows-with-visual-studio)

1. install [cmake](https://cmake.org/)
2. install virtual studio 2017 
3. download mongoc
4. build
```
cd mongo-c-driver-1.12.0
cmake -G "virtual studio 15 2017"  -DCMAKE_INSTALL_PREFIX="C:\mongo-c-driver" -DCMAKE_PREFIX_PATH="C:\mongo-c-driver"
msbuild.exe ALL_BUILD.vcxproj
```

Actually, the last step was compiled by vs IDE, not by command line.

**Comment by felipenoris** [comment]
I could put your binaries on [mongo-c-driver-builder repo](https://github.com/felipenoris/mongo-c-driver-builder), but do you know if it is OK to redistribute binaries compiled with visual studio?

**Comment by AlphaFase** [comment]
Adding test cases is no problem at all.

[test](https://github.com/AlphaFase/Mongoc.jl/blob/master/test/test-win.jl)

system: win10 x64  
julia: 1.0.0

**Comment by felipenoris** [comment]
I'm trying to compile it myself, using MINGW, but it failed. I opened an issue at https://jira.mongodb.org/browse/CDRIVER-3082 .

**Comment by felipenoris** [comment]
An alternative is to use appveyor to compile using Visual Studio.

**Comment by felipenoris** [comment]
I was able to compile using appveyor's Visual Studio. But hooking to Mongoc is just a stream of endless errors. This is so annoying!

**Comment by felipenoris** [comment]
Closed by d499d4ae2ec1bc60c77790421abcc41c20b95a1f.
